src/adagrams.js

```diff
@@ -1,15 +1,163 @@
 export const drawLetters = () => {
-  // Implement this method for wave 1
+  const letterPool = {
+    A: 9,
+    B: 2,
+    C: 2,
+    D: 4,
+    E: 12,
+    F: 2,
+    G: 3,
+    H: 2,
+    I: 9,
+    J: 1,
+    K: 1,
+    L: 4,
+    M: 2,
+    N: 6,
+    O: 8,
+    P: 2,
+    Q: 1,
+    R: 6,
+    S: 4,
+    T: 6,
+    U: 4,
+    V: 2,
+    W: 2,
+    X: 1,
+    Y: 2,
+    Z: 1,
+  };
+
+  let letters = "";
+  for (const [letter, num] of Object.entries(letterPool)) {
+    letters += letter.repeat(num);
+  }
+
+  const randomlyHand = [];
+  const lettersCountDict = {
+    A: 0,
+    B: 0,
+    C: 0,
+    D: 0,
+    E: 0,
+    F: 0,
+    G: 0,
+    H: 0,
+    I: 0,
+    J: 0,
+    K: 0,
+    L: 0,
+    M: 0,
+    N: 0,
+    O: 0,
+    P: 0,
+    Q: 0,
+    R: 0,
+    S: 0,
+    T: 0,
+    U: 0,
+    V: 0,
+    W: 0,
+    X: 0,
+    Y: 0,
+    Z: 0,
+  };
+
+  while (randomlyHand.length < 10) {
```

Comment:
Prefer to reference the integer 10 with a variable. This makes your code more readable/self documenting because it clues other devs in to the meaning of `10`.

```suggestion
  const HAND_LENGTH = 10;
  while (randomlyHand.length < HAND_LENGTH) {
```